src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -270,3 +271,13 @@ This datepicker has a predefined initial value that can be used both for initial
     />
   </Story>
 </Preview>
+
+# Reset DatePicker (from parect component)
```

Comment:
please move this before knobs story, there is an unwritten rule for stories that we have. The last story is the story with all the variables(knobs)

Reply:
fixed 8b5b3b2b59991b3d8c2db406a34ea21576814f01